crypto/wolfssl/CMakeLists.txt

```diff
@@ -71,12 +65,14 @@ if(ENABLE_STATIC_LIB)
     ARCHIVE_OUTPUT_NAME ngtcp2_crypto_wolfssl${STATIC_LIB_SUFFIX}
     C_VISIBILITY_PRESET hidden
   )
-  target_compile_definitions(ngtcp2_crypto_wolfssl_static PUBLIC
-    "-DNGTCP2_STATICLIB")
```

Comment:
Why is this deleted? Is it necessary for this PR?

Reply:
it is not required, because this definition comes with dependency on  `ngtcp2_static`